README.md

```diff
@@ -28,9 +28,9 @@ First, navigate to the project directory and install the necessary packages usin
 2. **Configure Database Properties:**
 Copy the provided example configuration file to create your own configuration:
```

Comment:
Actually this is no longer needed, bc the user should download the config sample file from dashboard setup page and upload the updated config there, not directly in the file system. Please update the setup part to reflect this.